src/_pytest/config/__init__.py

```diff
@@ -1270,14 +1270,16 @@ def _validate_plugins(self) -> None:
         missing_plugins = []
         for required_plugin in required_plugins:
             try:
-                spec = Requirement(required_plugin)
+                req = Requirement(required_plugin)
```

Comment:
Took the freedom to rename this, as `spec` is very close to `specifier` (another thing in `packaging`, see `.specifier` below) which I found confusing.